sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/users/EditUserSteps.java

```diff
@@ -0,0 +1,224 @@
+/*
+ * SORMAS® - Surveillance Outbreak Response Management & Analysis System
+ * Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package org.sormas.e2etests.steps.web.application.users;
+
+import static org.sormas.e2etests.pages.application.users.EditUserPage.*;
+
+import cucumber.api.java8.En;
+import javax.inject.Inject;
+import org.assertj.core.api.SoftAssertions;
+import org.sormas.e2etests.helpers.WebDriverHelpers;
+import org.sormas.e2etests.pojo.User;
+
+public class EditUserSteps implements En {
+  private final WebDriverHelpers webDriverHelpers;
+  protected User user;
+
+  @Inject
+  public EditUserSteps(final WebDriverHelpers webDriverHelpers, final SoftAssertions softly) {
+    this.webDriverHelpers = webDriverHelpers;
+
+    When(
+        "^I check the created data is correctly displayed on Edit User Page for selected ([^\"]*)$",
+        (String role) -> {
+          user = collectUserData();
+
+          softly
```

Comment:
can we use Truth and assert 2 objects to reduce these lines? :D

Reply:
updated